deadcode.txt

```diff
@@ -0,0 +1,186 @@
+MK_FP removed
+_DOS4G_hook_fini removed
+_DOS4G_hook_init removed
```

Comment:
Nope. I will remove it. For the rest I think you should ask Tom. I usually accept all the PRs, even if exe etc. are checked in, because at some point in the future we anyhow have to perform a repository cleanup and remove all the large binary blobs from the Git repo (and either move them to LFS or remove them for good).

Reply:
Understood. On the weekend I will look at the work on the Editor and check what is necessary for it to work, maybe I can remove the debug directory. I will also implement a build config without the  xcopy, then I can remove the other .proj files. Clearly asset the copy in the build got annoying :)